presto-main/src/main/java/com/facebook/presto/execution/QueryInfo.java

```diff
@@ -128,6 +129,87 @@ public QueryInfo(
         this.self = self;
         this.fieldNames = ImmutableList.copyOf(fieldNames);
         this.query = query;
+        this.queryAbridged = query;
+        this.queryStats = queryStats;
```

Comment:
@wagnermarkd  `queryAbridged` field needs to be initialized. Instead of having two constructors, we could remove the original one, and modify the testclasses where it is being used.